conda/shell/condabin/_conda_activate.bat

```diff
@@ -16,4 +16,4 @@
 @DEL /F /Q "%_TEMP_SCRIPT_PATH%"
 @SET _TEMP_SCRIPT_PATH=
 @SET "PROMPT=%CONDA_PROMPT_MODIFIER%%PROMPT%"
-@IF DEFINED PYTHONIOENCODING chcp %PYTHONIOENCODING% > NUL
+@IF DEFINED PYTHONIOENCODING chcp %PYTHONIOENCODING:~2% > NUL
```

Comment:
This will fail when PYTHONIOENCODING has been set without `cp`, which could happen for people coming from different ecosystems and during upgrade procedures from old conda before @jjhelmus' change to new conda with this change.

Please use the same code I do in https://github.com/mingwandroid/conda/commit/b62f31af1bcda45193c64f94272cc017c046ae73#diff-19d07b1f2eb7bd8f0c8e967b228f57d2 namely:

```
@IF DEFINED PYTHONIOENCODING chcp %PYTHONIOENCODING:cp=% > NUL
```

Reply:
got that, 👌